<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="goldGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Gold</string>
    </property>
    <widget class="QPushButton" name="pushValidateGold">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>62</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editGold">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="goldCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gold total : 5000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SkillsGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Stats</string>
    </property>
    <widget class="QPushButton" name="combatButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Combat !</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skillsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Skills</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dodoButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="grosDodoButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gros dodo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="bourrin_GroupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Bourrin</string>
    </property>
    <widget class="QPushButton" name="statsButton_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="vieProgressBar_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vie</string>
     </property>
    </widget>
    <widget class="QLabel" name="vieIndicator_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>33</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000/1000</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mpProgressBar_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mpIndicator_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>150/150</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireIndicator_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>76</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="freezeIndicator_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="poisonIndicator_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="stunIndicator_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="mage_GroupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Mage</string>
    </property>
    <widget class="QPushButton" name="statsButton_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vie</string>
     </property>
    </widget>
    <widget class="QLabel" name="vieIndicator_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>33</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000/1000</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mpProgressBar_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mpIndicator_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>150/150</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireIndicator_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>76</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="freezeIndicator_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="poisonIndicator_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="stunIndicator_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="assassin_groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Assassin</string>
    </property>
    <widget class="QPushButton" name="statsButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="vieProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vie</string>
     </property>
    </widget>
    <widget class="QLabel" name="vieIndicator">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>33</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000/1000</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mpProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mpIndicator">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>150/150</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireIndicator">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>76</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="freezeIndicator">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="poisonIndicator">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="stunIndicator">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="chaman_groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Chaman</string>
    </property>
    <widget class="QPushButton" name="statsButton_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="vieProgressBar_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vie</string>
     </property>
    </widget>
    <widget class="QLabel" name="vieIndicator_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>33</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000/1000</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mpProgressBar_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mpIndicator_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>150/150</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireIndicator_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>76</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="freezeIndicator_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="poisonIndicator_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="stunIndicator_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Healer</string>
    </property>
    <widget class="QPushButton" name="statsButton_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="vieProgressBar_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vie</string>
     </property>
    </widget>
    <widget class="QLabel" name="vieIndicator_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>33</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000/1000</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mpProgressBar_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mpIndicator_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>150/150</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireIndicator_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>76</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="freezeIndicator_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="poisonIndicator_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="stunIndicator_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Archer</string>
    </property>
    <widget class="QPushButton" name="statsButton_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="vieProgressBar_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vie</string>
     </property>
    </widget>
    <widget class="QLabel" name="vieIndicator_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>33</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000/1000</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mpProgressBar_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mpIndicator_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>150/150</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireIndicator_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>76</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="freezeIndicator_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="poisonIndicator_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="stunIndicator_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>76</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
